### ISSUE TITLE
maintenance 11.X and 12.0 are unable to build

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#7484]
**Describe the bug**
Same error 
- 12.0: https://gitlab.com/inverse-inc/packetfence/-/jobs/3685974495
- 11.2: https://gitlab.com/inverse-inc/packetfence/-/jobs/3685967709

Build passed on 2023/01/29.

Issue seems to be related to a change in `grunt` support inside npm.

**Comment by nqb** [comment]
On 12.0 failed job (compared to successful job):

```
npm WARN notsup Unsupported engine for grunt@1.6.0: wanted: {"node":">=16"} (current: {"node":"12.22.12","npm":"6.14.16"})
npm WARN notsup Not compatible with your version of node/npm: grunt@1.6.0
npm WARN notsup Unsupported engine for rimraf@4.1.2: wanted: {"node":">=14"} (current: {"node":"12.22.12","npm":"6.14.16"})
npm WARN notsup Not compatible with your version of node/npm: rimraf@4.1.2
```

**Comment by nqb** [comment]
Certainly caused by new release of [grunt](https://github.com/gruntjs/grunt/releases/tag/v1.6.0)

11.2 and 12.0 are trying to use latest grunt version where 12.1 is using 1.4.3.
I will see with @satkunas but I suppose it's because 12.1 has a `package-lock.json` in `html/common` but not 12.0 and 11.2

**Comment by nqb** [comment]
Test with https://github.com/inverse-inc/packetfence/compare/maintenance/12.0...fix/7484-maintenance-12-0.diff

**Comment by satkunas** [comment]
We're currently using Node 14 for the admin UI. If we bump the node version to >=14, does grunt@1.6.0 still complain?

**Comment by nqb** [comment]
I'm closing this because Grunt project released 1.6.1 as a fix: it doesn't require anymore to have node >= 16.
See https://github.com/gruntjs/grunt/issues/1752

We need to be careful on this in the future. I will add this as an R&D item.
